guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -333,10 +315,10 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
 
     });
 
-    $scope.$watch('menuShown', function setKeyboardEnabled(menuShown, menuShownPreviousState) {
+    $scope.$watch('menuShown', function menuVisibilityChanged(menuShown, menuShownPreviousState) {
         
         // Send clipboard data if menu is hidden
-        if (!menuShown && menuShownPreviousState)
+        if (!menuShown && menuShownPreviousState && angular.isString($scope.client.clipboardData))
```

Comment:
Actually, if we init ```client.clipboardData``` properly, this won't be necessary, I think. Let me try that, too...